### ISSUE TITLE
Liberica Native Image with File Chooser

### ISSUE THREAD

**Issue by java71** [bell-sw/Liberica#99]
Hi,
I use LIBERICA NATIVE IMAGE KIT with great success but I have a problem.
Do anyone manage to perform native-image with JavaFX file chooser.
It is impossible for me !!!

Maybe it is not supported at this stage ?
Any help is welcome !

--------------------------------------------------------------------------
```
glass.lib(CommonDialogs_Standard.obj) : error LNK2019: symbole externe non résolu __imp_GetOpenFileNameW référencé dans la fonction "class _jobject * __cdecl StandardFileChooser_Show(struct HWND__ *,wchar_t const *,wchar_t const *,wchar_t const *,long,unsigned char,class _jobjectArray *,long)" (?StandardFileChooser_Show@@YAPEAV_jobject@@PEAUHWND__@@PEB_W11JEPEAV_jobjectArray@@J@Z)
glass.lib(CommonDialogs_Standard.obj) : error LNK2019: symbole externe non résolu __imp_GetSaveFileNameW référencé dans la fonction "class _jobject * __cdecl StandardFileChooser_Show(struct HWND__ *,wchar_t const *,wchar_t const *,wchar_t const *,long,unsigned char,class _jobjectArray *,long)" (?StandardFileChooser_Show@@YAPEAV_jobject@@PEAUHWND__@@PEB_W11JEPEAV_jobjectArray@@J@Z)
c:\_data\CROSSWORD.Cli.Fr.exe : fatal error LNK1120: 2 externes non résolus
        at com.oracle.svm.hosted.image.NativeImageViaCC.handleLinkerFailure(NativeImageViaCC.java:509)
        at com.oracle.svm.hosted.image.NativeImageViaCC.write(NativeImageViaCC.java:456)
        at com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:695)
        at com.oracle.svm.hosted.NativeImageGenerator.run(NativeImageGenerator.java:515)
        at com.oracle.svm.hosted.NativeImageGeneratorRunner.buildImage(NativeImageGeneratorRunner.java:407)
        at com.oracle.svm.hosted.NativeImageGeneratorRunner.build(NativeImageGeneratorRunner.java:585)
        at com.oracle.svm.hosted.NativeImageGeneratorRunner.main(NativeImageGeneratorRunner.java:128)
        at com.oracle.svm.hosted.NativeImageGeneratorRunner$JDK9Plus.main(NativeImageGeneratorRunner.java:615)
```

**Comment by petermz** [comment]
This is a bug which I hope to fix this month. In the meantime, a workaround is to add the missing library to linker options: `-H:NativeLinkerOption=Comdlg32.lib`

**Comment by petermz** [comment]
Fixed in version 22.3.
- A few classes were made automatically configured for reflection access: `com.sun.glass.ui.mac.MacCommonDialogs` in Darwin, `com.sun.glass.ui.CommonDialogs` in Windows.
- For Windows, comdlg32.lib was added to the list of libraries.
